libp2p/Host.cpp

```diff
@@ -488,6 +497,11 @@ std::unordered_map<Public, std::string> Host::pocHosts()
 
 void Host::registerCapability(std::shared_ptr<CapabilityFace> const& _cap)
 {
+    if (haveNetwork())
```

Comment:
I believe this check should be in the other `registerCapability()` method which is called from this one.